packages/mdc-select/index.js

```diff
@@ -150,14 +175,49 @@ class MDCSelect extends MDCComponent {
     }
   }
 
+  /**
+   * Handles setup for the enhanced menu.
+   * @private
+   */
+  enhancedSelectSetup_(menuFactory) {
+    const isDisabled = this.root_.classList.contains(cssClasses.DISABLED);
+    this.selectedText_.setAttribute('tabindex', isDisabled ? '-1' : '0');
+    this.menuElement_ = /** @type {HTMLElement} */ (this.root_.querySelector(strings.MENU_SELECTOR));
+    this.menu_ = menuFactory(this.menuElement_);
+    this.menu_.hoistMenuToBody();
+    this.menu_.setAnchorElement(/** @type {!HTMLElement} */ (this.root_));
+    this.menu_.setAnchorCorner(menuSurfaceConstants.Corner.BOTTOM_START);
+    this.menu_.listen(menuSurfaceConstants.strings.CLOSED_EVENT, () => {
+      // menuOpened_ is used to track the state of the menu opening or closing since the menu.open function
+      // will return false if the menu is still closing and this method listens to the closed event which
+      // occurs after the menu is already closed.
+      this.menuOpened_ = false;
+      if (document.activeElement !== this.selectedText_) {
+        this.foundation_.handleBlur();
+      }
+    });
+
+    // Menu should open to the last selected element.
+    this.menu_.listen(menuSurfaceConstants.strings.OPENED_EVENT, () => {
+      if (this.selectedIndex >= 0) {
+        this.menu_.items[this.selectedIndex].focus();
+      }
+    });
+
+    this.menu_.listen(menuConstants.strings.SELECTED_EVENT, (evtData) => {
+      this.selectedIndex = evtData.detail.index;
```

Comment:
Can we add a comment here that explains that this ultimately runs `handleChange`? Otherwise it requires tracing 2 method calls deep to realize how the change event ends up emitted.